Library/ViewModels/RootViewModel.swift

```diff
@@ -457,7 +470,12 @@ private func tabData(forUser user: User?) -> TabBarItemsData {
 extension TabBarItemsData: Equatable {}
 extension TabBarItem: Equatable {}
 
-private func activitiesBadgeValue(with value: Int?) -> String? {
+private func activitiesBadgeValue(with value: Int?, hasPPOAction: Bool) -> String? {
```

Comment:
I like this interpretation, but my understanding was that if PPO is enabled, we never want to show a count. So ex if I have PPO enabled, no PPO actions, but I do have activity notifications, I was expecting the dot to show with no count. Would you mind checking with Alison what she intended before you submit this?

Reply:
Alison says we should show the count in the case where PPO is enabled, there is no PPO action, and there is activity count.